src/components/inputs/date-input/date-input.js

```diff
@@ -215,7 +216,11 @@ const DateInput = props => {
                         key={item}
                         isToday={isSameDay(today, item)}
                         {...getItemProps({
-                          disabled: props.isDisabled,
+                          disabled: !isDateInRange(
```

Comment:
Nit: `getIsDateInRange` is nicer when it comes then to distinguish between a boolean and the respective getter.